src/django/api/aws_batch.py

```diff
@@ -82,58 +88,66 @@ def submit_job(action, is_array=False, depends_on=None):
             raise RuntimeError(
                 'Failed to submit job {0}. Response {1}'.format(job_name, job))
 
+    def append_processing_result(result_dict):
+        query = ("UPDATE {item_table} "
```

Comment:
We are not building our query with any user submitted data, so we are not opening ourselves up to SQL injection.